llms/mlx_lm/lora.py

```diff
@@ -43,6 +43,26 @@
     "model": "mlx_model",
     "train": False,
     "fine_tune_type": "lora",
+    "optimizer": "adam",
+    "optimizer_config": {
+        "adam": {
+            "betas": [0.9, 0.999],
+            "eps": 1e-8,
```

Comment:
See comment below about only setting parameters here which don't have defaults for a given optimizer. Most of them should have defaults.